java/buildconf/ivy/obs-maven-config.yaml

```diff
@@ -79,7 +76,7 @@ artifacts:
   - artifact: commons-logging
     package: apache-commons-logging
     jar: apache-commons-logging\.jar
-    repository: Leap
+    repository: Leap_sle
   - artifact: commons-validator
     package: apache-commons-validator
```

Comment:
https://smelt.suse.de/maintained/?q=apache-commons-validator

I don't see it in the channels and I don't see it in the SLE codestreams either. you can make the switch if that source is somehow available in SLE. 

Am I missing something?

Reply:
yes, the line. The package is `apache-commons-logging` not `apache-commons-logging`.  https://smelt.suse.de/maintained/?q=apache-commons-logging